awx/main/models/workflow.py

```diff
@@ -650,6 +650,11 @@ class Meta:
         default=None,
         on_delete=models.SET_NULL,
     )
+    timeout = models.IntegerField(
+        blank=True,
+        default=0,
+        help_text=_("The amount of time (in seconds) before the approval node expires and fails."),
+    )
```

Comment:
This new timeout field and the change to `_get_unified_job_field_names` looks right to me.